.github/workflows/ci.yml

```diff
@@ -65,13 +65,13 @@ jobs:
       shell: bash {0}
       run: |
         git ls-remote --heads --exit-code https://github.com/NREL/lk.git $GIT_BRANCH
-        if [[ $? != "0" ]]; then echo "LK_BRANCH=develop" >> $GITHUB_ENV; else echo "LK_BRANCH=$GIT_BRANCH" >> $GITHUB_ENV; fi
+        if [[ $? != "0" ]]; then echo "LK_BRANCH=patch" >> $GITHUB_ENV; else echo "LK_BRANCH=$GIT_BRANCH" >> $GITHUB_ENV; fi
```

Comment:
We had a bit of a mess where someone branched from develop into patch, which required another commit to set these values to patch, and now we need yet another commit to set these to develop again. I think the correct steps are:

(1) Create a branch off of develop
(2) Merge patch into that branch
(3) Correct these branch names in a new commit
(4) Merge that branch into develop

I'll close this PR, do 1-3, and then send it to you for approval.

Reply:
Thanks for this - maybe docHub?